docs/functions/data.rst

```diff
@@ -99,6 +99,23 @@ Environment
    .. versionchanged:: 2.0
       Return method changed from stdout.
 
+.. function:: _lp_container() -> var:lp_container
+
+   Returns a string matching the container context of the shell. Valid values:
+
+   * ``Sing`` - running in a Singularity container
+   * ``Pdmn`` - running in a Podman container
+   * ``Dokr`` - running in a Docker container
+   * ``LXC`` - running in an LXC container
+
+   It is not possible for more than one context to be returned. The contexts
+   are checked in the order listed above, and the first one found will be
+   returned.
+
+   It is not possible for no context to be returned.
```

Comment:
I don't understand what this means. Obviously it is possible for nothing to be returned.

This function description needs a "Returns \`true\` if" section as well.

Reply:
I will fix this wording.